quint/src/repl.ts

```diff
@@ -327,7 +327,9 @@ export function quintRepl(
     const g = (s: string): string => {
       return chalk.gray(s)
     }
-    if (!line.startsWith('.')) {
+    // NB. the continue promppt is handled by `nextLine` as the input may be a
+    // copy and past from the reply itself.
```

Comment:
I had to google what `NB` was :sweat_smile: this is definitely not common use in this code base, so maybe it's better not to introduce it by now.

The other suggestions are on typos.
```suggestion
    // The continue prompt is handled by `nextLine` as the input may be a
    // copy and paste from the reply itself.
```